actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -221,6 +221,8 @@ def sent?;      synchronize { @sent };      end
 
     # Sets the HTTP status code.
     def status=(status)
+      raise ArgumentError, "#{status} is not a valid response code identifier. See Rack::Utils::SYMBOL_TO_STATUS_CODE "\
+        "for valid values" unless !status.is_a?(Symbol) || Rack::Utils::SYMBOL_TO_STATUS_CODE.include? status
```

Comment:
why it's not?

```
raise ArgumentError, "#{status} is not a valid response code identifier. See Rack::Utils::SYMBOL_TO_STATUS_CODE "\
        "for valid values" if status.is_a?(Symbol) && !Rack::Utils::SYMBOL_TO_STATUS_CODE.include?(status)
```

or create private method to make it easier to read
```
raise ArgumentError, "#{status} is not a valid response code identifier. See Rack::Utils::SYMBOL_TO_STATUS_CODE "\
        "for valid values" unless valid_http_status?(status)
```

```
def valid_http_status?(status)
  status.is_a?(Symbol) && Rack::Utils::SYMBOL_TO_STATUS_CODE.include?(status)
end
```

Reply:
Thanks for the feedback! The reason I used a not on `status.is_a?` is because you can pass an integer directly into this method. I've taken your advice of placing it in a private method.